src/utils.js

```diff
@@ -17,27 +17,55 @@ const exec = async (command, opts) => {
 
 const setupCml = async opts => {
   const { version, sudo = true } = opts;
+  const overrideRegex = /^git(?:\+(?:ssh|https))?:\/\//;
+  const orFlag = overrideRegex.test(version);
```

Comment:
slightly better name? e.g.

```suggestion
  const versionUrl = overrideRegex.test(version);
```

Reply:
that works for me